<?xml version='1.0'?>
<launch>

  <!-- Launch joystick ROS drivers -->
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="joy_topic" default="joy" />
  
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.3" />
    <param name="autorepeat_rate" value="20" />
    <remap from="joy" to="$(arg joy_topic)" />
  </node>

  <!-- Launch rosserial -->
  <!--<node pkg="rosserial_python" type="serial_node.py" name="serial_node_1" args="/dev/ttyUSB2" output="screen" /> -->
  <!--<node pkg="rosserial_python" type="serial_node.py" name="serial_node_2" args="/dev/ttyUSB0" output="screen" /> -->  
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node_4" args="/dev/ttyUSB0" output="screen" />  
  <!--<node pkg="rosserial_python" type="serial_node.py" name="serial_node_5" args="/dev/ttyUSB0" output="screen" /> -->

  <!-- Launch joint control -->
  <node pkg="zeus_arm" type="joint_teleop.py" name="joint_teleop_arm" output="screen">
  </node>



</launch>
